core/math/convex_hull.cpp

```diff
@@ -2278,9 +2279,15 @@ Error ConvexHullComputer::convex_hull(const Vector<Vector3> &p_points, Geometry3
 			e = e->get_next_edge_of_face();
 		} while (e != e_start);
 
+		// reverse indices: Godot wants clockwise, but this is counter-clockwise
+		if (face.indices.size() > 2) {
+			int *indices = face.indices.ptrw();
+			std::reverse(&indices[1], &indices[face.indices.size()]); // reverse all but the first index.
+		}
```

Comment:
This is fine now .. only thing is I don't know whether or not `std::reverse` is allowed in Godot core (@akien-mga will be able to clarify):
https://docs.godotengine.org/en/stable/about/faq.html#why-does-godot-not-use-stl-standard-template-library